src/Module.ts

```diff
@@ -998,12 +1000,19 @@ export default class Module {
 		this.dynamicImports.push({ argument, id: null, node, resolution: null });
 	}
 
+	private hasExportNamedAs(name: string) {
+		return this.exports.has(name) || this.reexportDescriptions.has(name);
+	}
```

Comment:
Considering that this function is always used like

```js
if (this.hasExportNamedAs(foo)) {
  this.error(logDuplicateExportError(foo), nodeStart);
}
```

maybe move the if-statement into this call:

```suggestion
	private assertUniqueExportName(name: string, nodeStart: number) {
	  if (this.exports.has(name) || this.reexportDescriptions.has(name)) {
	    this.error(logDuplicateExportError(name), nodeStart);
	  };
	}
```

and adapt all call sites.